trees/app/src/main/java/com/trees/common/pyi/ImageProcessor.java

```diff
@@ -57,6 +66,13 @@ public ImageResult processImage(Activity context, ImageRaw raw) {
                 Toast.makeText(context, R.string.noDepthPoints, Toast.LENGTH_LONG).show();
             }
 
+            // log the info
+            Log.i("Miranda", "Log info:" + logInfo);
```

Comment:
Could you clean this up to use a standard log string? I know there are other places hanging around where I do this with "Amelia" ... but better to move slowly towards a better code style :)

Reply:
Sure, will do!

Reply:
I changed the logtag to "IntermediateData", is that what you are suggesting? Please correct me if I've misconstrued. Also, would you like me to modify all occurrences of those non-standard log strings?